scripts/stats.js

```diff
@@ -0,0 +1,24 @@
+var stats = {};
+var statData = '';
+
+stats.getData = function () {
+  $.getJSON('scripts/hackerIpsum.json', function(data) {
+    statData = data;
```

Comment:
suggestion to make this stats.statData (add it as an array onto your stats object), then you can work with it outside the stats.getData function.